app/services/proofing/aamva/proofer.rb

```diff
@@ -36,7 +36,7 @@ def proof(applicant)
         )
         build_result_from_response(response)
       rescue => exception
-        NewRelic::Agent.notice_error(exception)
+        NewRelic::Agent.notice_error(exception) unless exception.is_a? Proofing::TimeoutError
```

Comment:
Remind me how we'll notice if everything is timing out at once? Will this affect the alert that John Skinner put in place?

Reply:
Follow-up question, why do we still want to log timeout errors to Newrelic? Are those the actionable ones?

Reply:
wait sorry I got the logic flipped (this is why I hate `unless``!! separate issue), wouldn't we want to stop logging basically *all* errors to NewRelic to cut down on noise?